app/data/battlepets.json

```diff
@@ -702,6 +702,48 @@
         ],
         "name": "Operation: Mechagon"
       },
+      {
+        "id": "dbddd3a4",
+        "items": [
+          {
+            "creatureId": 162499,
+            "icon": "inv_icon_wing03d",
+            "name": "Oasis Void-Duster",
+            "spellid": null
+          },
+          {
+            "creatureId": 154796,
```

Comment:
This is the creature ID of the Clanking Scrapsorter, not the Void-Scarred Locust.

Reply:
Fixed.